Selection_Sort/SelectionSort.c

```diff
@@ -0,0 +1,74 @@
+//C program to implement Selection Sort
+ 
+  #include <stdio.h>
+  #include <conio.h>
+  
+       //function to swap elements
+  
+		void swap(int *ele1, int *ele2)
+		{
+			int temp = *ele1;
+			*ele1 = *ele2;
+			*ele2 = temp;
+		}
+		
+		//function to perform selection sort
+		
+		void SelectionSort(int arr[], int len)
+		{
+			int i, j, min;
+			
+			//finding minimum element in unsorted array
+			
+			  for(i = 0; i < len-1; i++)
+			  {
+			  	min = i;
+			  	
+			  	 for(j = i+1; j < len; j++)
+			  	 {
+			  	 	if(arr[j] < arr[min])
+			  	 	   min = j;
+			  	 	   
+				   }
+				 
+				 //swapping minimum element with first element
+				 
+				 swap(&arr[min], &arr[i]);
+				   
+			  }
+		}
+		
+		//function to display elements of an array
+		
+		void displayArray(int arr[], int size) 
+         { 
+         
+            for (int i=0; i < size; i++) 
+              printf("%d ", arr[i]); 
+              
+        printf("\n"); 
+        } 
+        
+        //main function to control program execution by calling other functions
+        
+        int main()
+        {
+        	int ar[10], length;
```

Comment:
increase the array size 
if possible make it dynamic or depending upon user input

Reply:
I have updated the code. The size of the array now depends on user input. Please check it out.